.travis.yml

```diff
@@ -62,10 +61,35 @@ jobs:
           repo: pytest-dev/pytest
     - stage: linting
       python: '3.6'
-      env: TOXENV=linting
+      env: TOXENV=linting PYTEST_NO_COVERAGE=1
+
+before_script:
+  - |
+    if [[ "$PYTEST_NO_COVERAGE" != 1 ]]; then
+      export _PYTEST_TOX_COVERAGE_RUN="env COVERAGE_FILE=$PWD/.coverage coverage run --source {envsitepackagesdir}/_pytest/,$PWD/testing -m"
+      export _PYTEST_TOX_EXTRA_DEP=coverage-enable-subprocess
+      export COVERAGE_PROCESS_START="$PWD/.coveragerc"
+    fi
 
 script: tox --recreate
 
+after_success:
+  - |
+    if [[ "$PYTEST_NO_COVERAGE" != 1 ]]; then
+      set -e
+      pip install codecov
+      coverage combine
+      coverage xml
+      coverage report -m
+      codecov --required -X gcov pycov search -f coverage.xml --flags ${TOXENV//-/ }
+
+      # Coveralls does not support merged reports.
```

Comment:
I believe we can drop coveralls then. Or would you rather do it in another PR @blueyed?

Reply:
I've thought to keep it for comparison.

Reply:
Oh OK. But I think we should remove it after a while to avoid confusion. 👍